tests/release/const.go

```diff
@@ -18,6 +17,35 @@ const (
 	environment                         = "test-environment"
 	releaseStrategyServiceAccount       = "pipeline"
 
+	sourceReleaseLinkName                string = "source-release-link"
```

Comment:
releaselink no longer exists. It is releaseplan and releaseplanadmission

Reply:
removed